### ISSUE TITLE
Hide the visually hidden file input from assistive technologies

### ISSUE THREAD

**Issue by afercia** [moxiecode/moxie#185]
On a related issue on the WordPress Trac, it was pointed out that the visually hidden input of type=file `FileInput` is still perceived by assistive technologies e.g. screen readers. See https://core.trac.wordpress.org/ticket/47611

This input is visually hidden (positioned off screen) thus it's meant to be not available to sighted users.

A while ago, it was also made not available to keyboard users by adding a `tabindex="-1"` attribute to it, see https://github.com/moxiecode/moxie/commit/d2e3bfdf8fbf5e1674fae09db445af811ae36c61

However, when using a screen reader this input is still perceived and can be navigated to, for example:
- by using arrow keys (which is the most commonly navigation mechanism used by screen reader users)
- by using one of the several navigation tools screen readers provide, for example: the list of form controls in a page (see screenshot below)

In order to make this input non perceivable by assistive technology users too, it would be great to consider to add an `aria-hidden="true"` attribute to it. This should be enough to effectively hide it from assistive technologies (pending testing with various browsers / screen readers / other assistive technologies).

Screenshot using Safari + VoiceOver and the list of form controls provided by the VoiceOver "rotor" in WordPress:

<img width="1440" alt="Screenshot 2019-12-11 at 10 17 54" src="https://user-images.githubusercontent.com/1682452/70608963-4d8f3f80-1c01-11ea-80b1-274c294f19cb.png">

Screenshot using Safari + VoiceOver and Control Option Arrow keys to navigate content:

<img width="1440" alt="Screenshot 2019-12-11 at 10 29 30" src="https://user-images.githubusercontent.com/1682452/70608977-5122c680-1c01-11ea-8dd1-89fa2a236e13.png">
